db/migrate/20210715164138_add_name_and_image.rb

```diff
@@ -0,0 +1,22 @@
+class AddNameAndImage < ActiveRecord::Migration[6.0]
+  def change
+    add_column :cpus, :name, :string
+    add_column :cpus, :image, :string
+    add_column :mobos, :name, :string
+    add_column :mobos, :image, :string
+    add_column :gpus, :name, :string
+    add_column :gpus, :image, :string
+    add_column :psus, :name, :string
+    add_column :psus, :image, :string
+    add_column :rams, :name, :string
+    add_column :rams, :image, :string
+    add_column :storages, :name, :string
+    add_column :storages, :image, :string
+    add_column :cases, :name, :string
+    add_column :cases, :image, :string
+    add_column :case_fans, :name, :string
+    add_column :case_fans, :image, :string
+    add_column :cpu_fans, :name, :string
+    add_column :cpu_fans, :image, :string
+  end
```

Comment:
Have a separate migration file per table

Reply:
Our thought process for this since we have multiple tables and we're just adding the same columns, we agreed to just do it in one migration file. But moving forward we will do separate migration files for each table.

Reply:
Good day maam, sorry for the mistake. Will be more cautious moving forward.